internal/graphql/model/operator_helper.go

```diff
@@ -117,11 +122,12 @@ func GetMeshSyncInfo(mesheryclient operatorClient.Interface, mesheryKubeClient *
 	if err != nil && !kubeerror.IsNotFound(err) {
 		return meshsyncStatus, ErrMesheryClient(err)
 	}
-
-	meshsyncDeployment, err := mesheryKubeClient.KubeClient.AppsV1().Deployments("meshery").Get(context.TODO(), "meshery-meshsync", metav1.GetOptions{})
-	meshsyncVersion := ""
-	if err == nil {
-		meshsyncVersion = imageVersionExtractUtil(meshsyncDeployment.Spec.Template, "meshsync")
+	meshsyncVersion, err := getVersion(mesheryKubeClient)
+	if meshsyncVersion == "" {
+		meshsyncVersion = "latest"
```

Comment:
Can we also retrieve the associated version (git) tag? e.g. `stable-v0.5.19`?

Reply:
Yes.

Reply:
Good. We’ll want to send back the specific version tag as opposed to `latest`

Reply:
In any case, MeshSync is not running or not started, instead of showing nothing (blank), latest will be shown first-time and when again MeshSync starts successfully actual version tag will be visible.